src/components/openSourceDep/DashboardOpenSourceDependencies.vue

```diff
@@ -79,16 +81,156 @@
             </stats-card>
           </b-col>
       </b-row>
+      <!-- Language Counts -->
+      <b-row>
+        <b-col>
+          <stats-card>
+            <div slot="header" class="icon-success">
+              <b-icon icon="arrow-right" class="con-big text-center" style="color: green;"></b-icon>
```

Comment:
It would be great to get the NodeJS logo from the public folder here (might need to add it first)